tools/cldr-apps/js/test/run.sh

```diff
@@ -4,6 +4,6 @@
 # Optionally, to run this as cldrtestjs, you may have in .bash_profile:
 # alias cldrtestjs='cd [path/to/cldr] && sh tools/cldr-apps/js-unittest/run.sh'
 cd tools/cldr-apps/js
-npm run test
+npm run build-test
```

Comment:
this still works, but you can also run `npm t` to do a command line run

Reply:
awesome!